automated/linux/systemd-analyze/systemd-analyze.yaml

```diff
@@ -0,0 +1,29 @@
+# SPDX-License-Identifier: GPL-2.0-only
+# Copyright (C) 2024 Linaro Ltd.
+metadata:
+    format: Lava-Test Test Definition 1.0
+    name: systemd-analyze
+    description: |
+        The systemd-analyze analyzes the boot times. It's boot times
```

Comment:
*measures*

Reply:
Thank you @bhcopeland this PR https://github.com/Linaro/test-definitions/pull/508 should fix it